### DISCUSSION TITLE
How about replace `Axios` with `ofetch`?

### DISCUSSION THREAD

**Discussion by winner106** [BearStudio/start-ui-web#390] [General]
[https://github.com/unjs/ofetch](https://github.com/unjs/ofetch)

**Comment by yoannfleurydev** [comment]
Hi @winner106 , it is always a pleasure to see that you are still interested in Start UI [web]!

Actually, we are thinking and working on the data fetch part on our side, we are checking multiple solutions at the moment. We are thinking about replacing the current TanStack Query + Axios part with something like ts-rest + another fetch library to enjoy better typings and developer experience.

Did you try `ofetch`? What do you think about it?
